parkhangs-webapp/src/components/GoogleButtons.js

```diff
@@ -0,0 +1,93 @@
+import React, {Component} from 'react'
+import {connect} from 'react-redux'
+import {GoogleLogin, GoogleLogout} from 'react-google-login';
+import {
+    getUser,
+    setAccessToken,
+    logout
+} from 'features/users/userSlice'
+
+
+const CLIENT_ID = process.env.REACT_APP_GOOGLE_API_CLIENT_ID;
+
+class GoogleButtons extends Component {
+
+    constructor(props) {
+        super(props);
+
+        this.login = this.login.bind(this);
+        this.handleLoginFailure = this.handleLoginFailure.bind(this);
+        this.handleLogoutFailure = this.handleLogoutFailure.bind(this);
+    }
+
+    login(response) {
+
+        if (response.accessToken) {
+
+            const firstName = response.profileObj.givenName;
+            const lastName = response.profileObj.familyName;
+            const email = response.profileObj.email;
+            const accessToken = response.accessToken; // TODO: could be use for Google Calendar integration
```

Comment:
We can attach the token to the google calendar API session for security later on, you mean?

Reply:
I removed it. not used right now